pkg/operator/loglevel/logging_controller.go

```diff
@@ -51,6 +51,11 @@ func (c LogLevelController) sync(ctx context.Context, syncCtx factory.SyncContex
 		desiredLogLevel = c.defaultLogLevel
 	}
 
+	if !ValidLogLevel(desiredLogLevel) {
+		syncCtx.Recorder().Warningf("OperatorLogLevelInvalid", "Invalid logLevel %q, falling back to %q", desiredLogLevel, c.defaultLogLevel)
+		desiredLogLevel = c.defaultLogLevel
```

Comment:
@deads2k Jordan was working on this mechanism to get warning strings to the user (kubectl), used for deprecation warnings in his case. Is that live already? Could it be used via admission to tell the user that the loglevel is invalid without failing?

Reply:
> @deads2k Jordan was working on this mechanism to get warning strings to the user (kubectl), used for deprecation warnings in his case. Is that live already? Could it be used via admission to tell the user that the loglevel is invalid without failing?

Maybe.  I guess I would want to just force clear bad data one release.